objects/rough-work/js/array-of-objects.js

```diff
@@ -0,0 +1,64 @@
+// 1. Create an array to hold quiz questions and answers
+const questions = [
+    {
+        question: 'How many planets are in the Solar System?',
+        answer: '8'
+    },
+    {
+        question: 'How many continents are there?',
+        answer: '7'
+    },
+    {
+        question: 'How many legs does an insect have?',
+        answer: '6'
+    },
+    {
+        question: 'What year was JavaScript created?',
+        answer: '1995'
+    }
+];
+
+// 2. Store the number of questions answered correctly
+const correct = [];
+const incorrect = [];
+let correctAnswers = 0;
+
+/* 
+  3. Use a loop to cycle through each question
+      - Present each question to the user
+      - Compare the user's response to answer in the array
+      - If the response matches the answer, the number of correctly
+        answered questions increments by 1
+*/
+for (let i = 0; i < questions.length; i++) {
+    let question = questions[i].question;
+    let answer = questions[i].answer;
+    let response = prompt(question);
+
+    if (response === answer) {
+        correctAnswers++;
+        correct.push(question);
+    } else {
+        incorrect.push(question);
+    }
```

Comment:
Have you tried this one yet?
```
if (response === answer) {
    correctAnswers++;
    correct.push(question);
    continue;
}
incorrect.push(question);
```

Reply:
Yes I have tried the continue, it works great! Just following the instructor in my rough work.